.github/workflows/devcontainer.yml

```
@@ -0,0 +1,137 @@
+
+jobs:
+  build:
+    name: Build Devcontainer Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v6
+
+      - name: Set registry
+        id: registry
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:5:30: Double quote to prevent globbing and word splitting

(shellcheck)